Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -131,6 +131,11 @@
 # "Export" Section
 reconstruction         = cms.Sequence(localreco*globalreco*highlevelreco*logErrorHarvester)
 
+#logErrorHarvester should only wait for items produced in the reconstruction sequence
+_modulesInReconstruction = list()
+reconstruction.visit(cms.ModuleNamesFromGlobalsVisitor(globals(),_modulesInReconstruction))
+logErrorHarvester.includeModules = cms.untracked.vstring(set(_modulesInReconstruction))
```

Comment:
what should I do to make logErrorHarvester also collect inputs from other 
- tasks
- sequences
?

Reply:
The exact same pattern would work on `Tasks` or `Sequences` since both of those types support the _visitor_  pattern. One difference would be that instead of setting `logErrorHarvester.includeModules` you'd probably want to extend it
```python
logErrorHarvester.includeModules = cms.untracked.vstring( set(logErrorHarvester.includeModules).union(another_list_of_modules) )
```